src/js/view/view.js

```diff
@@ -118,6 +118,15 @@ function View(_api, _model) {
 
         // Don't update container dimensions to 0, 0 when not in DOM
         if (containerWidth || containerHeight || inDOM) {
+            if (!_resizeOnFloat && floatingPlayer) {
+                // Update floating player based on player elements bounds when updateBounds fires on orientationChange.
+                const { floatingWidth, floatingHeight } = _getFloatingDimensions(containerWidth, containerHeight);
+                containerWidth = floatingWidth;
+                containerHeight = floatingHeight;
+                _resizeOnFloat = true;
+                _resizePlayer(floatingWidth, floatingHeight, true);
```

Comment:
`updateBounds` purpose is just that: to update "containerWidth" and "containerHeight" . It should not resize the player. That should have already happened before `updateBounds`  is called.

Reply:
Can't you just add min/max-width and height  styles to the player in `_updateFloating`:

```js
style(_playerElement, {
    backgroundImage: _preview.el.style.backgroundImage || _model.get('image'),
    maxWidth, // floatingWidth
    maxHeight // floatingHeight
});
```

`updateBounds` should be called by the responsive listener and update the player dimensions after it's floated. We shouldn't need to resize the view at all.